web/src/beta/features/Editor/Story/PagesPanel/index.tsx

```diff
@@ -1,135 +1,103 @@
-            uniqueKey="LeftPanelPages"
-            gap={8}
-            items={items}
-            getId={item => item.id}
-            onItemDrop={async (item, index) => {
-              setItems(old => {
-                const items = [...old];
-                items.splice(
-                  old.findIndex(o => o.id === item.id),
-                  1,
-                );
-                items.splice(index, 0, item);
-                return items;
-              });
-              await handleStoryPageMove(item.id, index);
-            }}
-            renderItem={(storyPage, i) => {
-              const title = (getFieldValue(storyPage.property.items ?? [], "title", "title") ??
-                t("Untitled")) as string;
-              const hasEmptySpace = isEmptyString(title);
-              return (
-                <PageItemWrapper
-                  key={storyPage.id}
-                  pageCount={i + 1}
-                  isSwipeable={storyPage.swipeable}>
-                  <PageItem
-                    key={i}
-                    isSelected={selectedStoryPage?.id === storyPage.id}
-                    isOpenAction={openedPageId === storyPage.id}
-                    onItemClick={() => handleStoryPageSelect(storyPage.id)}
-                    onActionClick={() => setOpenedPageId(old => (old ? undefined : storyPage.id))}
-                    onOpenChange={isOpen => {
-                      setOpenedPageId(isOpen ? storyPage.id : undefined);
-                    }}
-                    onPageDelete={() => handleStoryPageDelete(storyPage.id)}
-                    title={hasEmptySpace || !title ? t("Untitled") : title}
-                    setOpenedPageId={setOpenedPageId}
-                    propertyId={storyPage.property.id}
-                    storyPage={storyPage}
-                    onPropertyUpdate={handlePropertyValueUpdate}
-                  />
-                </PageItemWrapper>
-              );
-            }}
-          />
-        </SContentUp>
-        <SContentBottom>
-          <Action
-            icon="square"
-            title={`+ ${t("New Page")}`}
-            onClick={() => handleStoryPageAdd(false)}
-          />
-          {/* <Action icon="swiper" title={`+ ${t("New Swipe")}`} onClick={() => onPageAdd(true)} /> */}
-        </SContentBottom>
-      </SContent>
+      <ButtonWrapper>
+        <Button
+          icon="plus"
+          title="New Page"
+          size="small"
+          extendWidth
+          onClick={() => handleStoryPageAdd(false)}
+        />
+      </ButtonWrapper>
+      <Wrapper onScroll={openedPageId ? () => setOpenedPageId(undefined) : undefined}>
+        <DragAndDropList
+          items={DraggableStoryPageItems}
+          handleClassName={PAGES_DRAG_HANDLE_CLASS_NAME}
+          onMoveEnd={handleMoveEnd}
+          onMoveStart={handleMoveStart}
+          dragDisabled={false}
+        />
+        <EmptySpace onClick={() => handleStoryPageSelect(undefined)} />
```

Comment:
```suggestion
        <EmptySpace onClick={() => handleStoryPageSelect(undefined)} />
```
Maybe we don't need this since there will always some page in active, unselect a story page seems impossible in our design, what do you think?